cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -141,6 +143,8 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Int64Var(&s.NodeMilliCPU, "node-milli-cpu", s.NodeMilliCPU, "The amount of MilliCPU provisioned on each node")
 	fs.Var(resource.NewQuantityFlagValue(&s.NodeMemory), "node-memory", "The amount of memory (in bytes) provisioned on each node")
 	fs.StringVar(&s.ClusterName, "cluster-name", s.ClusterName, "The instance prefix for the cluster")
+	fs.StringVar(&s.ClusterClassB, "cluster-class-b", "10.244", "Class B network address for Pods in cluster.")
```

Comment:
It should also work for /14. Please note that we use the third byte for identifying node and the last byte for identifying pods inside node. We use first two bytes for the whole network, so, if GKE uses /14 last two bits of the second bytes will be always set to 0.

If you insist on converting this to CIDR, I can do it in a separate PR, as this PR is already waiting much too long.